sys-cluster/genders/genders-1.27.3.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit autotools distutils-r1 perl-module java-pkg-opt-2
+
+case "${PV}" in
+9999)
+	SRC_URI=""
+	EGIT_REPO_URI="https://github.com/chaos/${PN}.git"
+	inherit git-r3
+	KEYWORDS=""
+	;;
+*)
+	MY_PV="$(ver_rs 1-2 -)"
+	MY_P="${PN}-${MY_PV}"
+	SRC_URI="https://github.com/chaos/${PN}/archive/${MY_P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-${MY_P}"
+	;;
+esac
+
+DESCRIPTION="Genders is a static cluster configuration database used for cluster configuration management."
+HOMEPAGE="https://github.com/chaos/genders"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="cxx java perl python"
+
+CDEPEND="
```

Comment:
Don't use `CDEPEND` as 1char+DEPEND should be reserved for internal use. Use `COMMON_DEPEND` instead.

Reply:
please write something in the devmanual about it before pissing people

Reply:
There are also 8000 lines of "EAPI 5" in the portage tree. You wouldn't use it now.

Bug, 
https://bugs.gentoo.org/392239

Discussion in the mailing list, 
https://archives.gentoo.org/gentoo-dev/message/db4986470a316a3653bfecd593d5007b